### PR TITLE
UI setup

### PR DESCRIPTION
**Pull request by supermanshuvo**
Dear Sir,
I think I have finished the ui design. Now I want to work on the next area hope you like the ui.
Thank you
by 
Rezaul Kaim

### CODE REVIEW

File: resources/views/contact/add.blade.php [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<!-- ******************************** -->
```
Comment (asifmalik106) [inline resources/views/contact/add.blade.php:1]: Remove all the  lines from the entire project

File: resources/views/contact/add.blade.php [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+<!-- ADD EXTENDS PHP LAYOUTS PROPERTY -->
```
Comment (asifmalik106) [inline resources/views/contact/add.blade.php:2]: for @extend, @section and related blade syntaxes use only single comment on start. No need to put end comment.

File: resources/views/contact/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+<!-- END Header Area -->
+<!-- ********************* -->
+    <div class="row mt-5">
+        <div class="col-xs-12 col-sm-12 col-md-4 col-lg-4 col-xl-4">
+            <div class="user">
+                <img src="img/usericon.png" class="img-fluid img-thumbnail mb-5" alt="There Is an User Icon">
+                <h3>User Name : SupermanShuvo</h3>
+            </div>
+            <div class="logout mt-5">
+                <a href="{{ url('/logout') }}"><input type="button" value="Logout" class="btn btn-warning"></a>
```
Comment (asifmalik106) [inline resources/views/contact/index.blade.php:31]: always use route name in this case

File: resources/views/contact/edit.blade.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    <!-- ********************* -->
+    <!-- START CARD -->
+    <!-- ********************* -->
+        <div class="card bg-style">
+            <div class="card-body">
+            <!-- ********************* -->
+            <!-- START Edit FORM -->
+            <!-- ********************* -->
+            <h1 class="h3 mb-3 font-weight-normal">Edit Contact</h1>
+            <a href="{{ url('/') }}" class="btn btn-primary">Go Back</a>
```
Comment (asifmalik106) [inline resources/views/contact/edit.blade.php:27]: always use route name

File: resources/views/layouts/app.blade.php [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <!-- END CSS LINK -->
+</head>
+
+<body>
+    <!-- ********************* -->
+    <!-- START NAVBAR -->
+    <!-- ********************* -->
+    <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
+      <a class="navbar-brand" href="{{ url('/') }}">{{ config('app.name')}}</a>
```
Comment (asifmalik106) [inline resources/views/layouts/app.blade.php:22]: always use route name

File: routes/web.php [hunk 6]
```diff
@@ -14,5 +14,17 @@
 */
```
Comment (asifmalik106) [inline routes/web.php:15]: always use route name for each and every route.

**Review by asifmalik106 — CHANGES_REQUESTED** [review]

File: resources/views/contact/add.blade.php (hunk 1 above)
Comment (supermanshuvo) [reply resources/views/contact/add.blade.php:1]: Done

File: resources/views/contact/index.blade.php (hunk 3 above)
Comment (supermanshuvo) [reply resources/views/contact/index.blade.php:31]: yes done

**Comment by asifmalik106** [comment]
http://www.laravelbestpractices.com/

File: resources/views/layouts/app.blade.php (hunk 5 above)
Comment (supermanshuvo) [reply resources/views/layouts/app.blade.php:22]: work is done

File: routes/web.php (hunk 6 above)
Comment (supermanshuvo) [reply routes/web.php:15]: work is done

File: resources/views/contact/edit.blade.php (hunk 4 above)
Comment (supermanshuvo) [reply resources/views/contact/edit.blade.php:27]: done now

**Comment by supermanshuvo** [comment]
working in progress

**Pull request closed without merging**